osd/Ingress-Controllers.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Ingress controller blocked, action required",
```

Comment:
Typical summary convention has been this pattern, "Action required: Update ingress controller configuration"